src/schemas/22-5490/schema.js

```diff
@@ -68,7 +70,7 @@ let schema = {
       type: 'string',
       enum: ['chapter35', 'chapter33']
     },
-    previousBenefits: {
+    previousBenefits: Object.assign({
```

Comment:
if there's a file that hasn't included lodash i can use `Object.assign` without including anything, and if i'm going to do that then i should use it everywhere for consistency